vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -1003,7 +1041,951 @@ func TestStandardTxExecutorDurangoAddValidator(t *testing.T) {
 	require.Equal(endTime, val.EndTime)
 }
 
+func TestDurangoStandardTxExecutorAddPermissionlessValidatorTx(t *testing.T) {
+	require := require.New(t)
+	env := newEnvironment(t, true /*=postBanff*/, true /*=postCortina*/, true /*=postDurango*/)
+	env.ctx.Lock.Lock()
+	defer func() {
+		require.NoError(shutdownEnvironment(env))
+		env.ctx.Lock.Unlock()
```

Comment:
I believe it does not really matter in this case, since this happens at the very end of the test. But this is something that can be fixed for all tests. https://github.com/ava-labs/avalanchego/pull/2491 by @dhrubabasu fixes this concern I believe.